src/MCPClient.Dockerfile

```diff
@@ -99,4 +99,4 @@ RUN mkdir -p /etc/archivematica && (echo "---"; \
 
 USER archivematica
 
-ENTRYPOINT /src/MCPClient/lib/archivematicaClient.py
+ENTRYPOINT ["/src/MCPClient/lib/archivematicaClient.py"]
```

Comment:
This is neat. The difference in meaning is subtle; there are probably other Docker images in our stack that could benefit from a change like this too.